gobblin-yarn/src/main/java/gobblin/yarn/ConfigurationConstants.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.yarn;
+
+/**
+ * A central place for configuration related constants of Gobblin on Yarn.
+ *
+ * @author ynli
+ */
+public class ConfigurationConstants {
```

Comment:
Can you rename to `YarnConfigurationKeys` so as to not be confused with `gobblin.configuration.ConfigurationKeys`.